commands/kubebuilder/kubebuilder.go

```diff
@@ -18,6 +18,37 @@ func AddCommands(cli *cli.Cli) {
 			fmt.Println("See help for working with kubebuilder")
 		},
 		SubCommands: []command.Command{
+			command.Command{
+				Name: "auth",
+				Help: "Authenticates you against all required services",
```

Comment:
Please add in a mention this is for GCP GCR

Reply:
Sorted